src/main/java/com/javapractices/app/App.java

```diff
@@ -4,10 +4,10 @@
  * Hello world!
  *
  */
-public class App 
-{
-    public static void main( String[] args )
-    {
-        System.out.println( "Hello World!" );
+public class App {
+    public static void main(String[] args) {
+        Account account = new Account();
+        System.out.println(account);
```

Comment:
This class hasn't toString method implemented. So It'll use the toString inherit from Object class.